### PR TITLE
feat: update auro-library to 3.0.1

### PR DESCRIPTION
**Pull request by DukeFerdinand**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #276 
## Summary:

This updates to the latest auro-library version and ensures the documentation paths are updated in `index.md` and `api.md` to match!

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [x] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Upgrade auro-library to version 3.0.1 and update documentation paths to match the new directory structure.

Enhancements:
- Update documentation paths in index.md and api.md to reflect new directory structure.

Build:
- Upgrade auro-library dependency to version 3.0.1.

Documentation:
- Update documentation paths in index.md and api.md to reflect new directory structure.

## Summary by Sourcery

Upgrade auro-library to version 3.0.1 and update documentation paths to match the new directory structure.

New Features:
- Upgrade auro-library dependency to version 3.0.1.

Enhancements:
- Update documentation paths in index.md and api.md to reflect new directory structure.

Documentation:
- Update documentation paths in index.md and api.md to reflect new directory structure.

### CODE REVIEW

File: demo/api.min.js [hunk 1]
```diff
@@ -861,6 +861,76 @@
...
+   */
+  closestElement(
+    selector, // selector like in .closest()
+    base = this, // extra functionality to skip a parent
+    __Closest = (el, found = el && el.closest(selector)) =>
+      !el || el === document || el === window
+        ? null // standard .closest() returns null for non-found selectors also
+        : found
+          ? found // found a selector INside this element
+          : __Closest(el.getRootNode().host) // recursion!! break out to parent DOM
```
[the remark this answers is not in the document]
Comment (DukeFerdinand) [reply demo/api.min.js:899]: same as other, out of scope

**Review by chrisfalaska — DISMISSED** [review]
Tested locally. 🚀

**Review by chrisfalaska — APPROVED** [review]
Tested locally 🚀

**Pull request merged**